### ISSUE TITLE
Using outdated api url "v1beta1"

### ISSUE THREAD

**Issue by riekusr** [serene-water/node-red-contrib-coda-io#10]
My node is not fetching rows, I think it is because of the api url, the node generates this url to fetch: 

https://coda.io/apis/v1beta1/docs/ZVPFKP0J/tables/grid-1uDUkK0P/rows?limit=500

that gets a 404, when you change the url from "v1beta1" to "v1" it works in my browser.

where do I adjust the base url?

**Comment by the-greenman** [comment]
Yes, I've been lazy and not updated this. I'll try to get a new release out
in a day or two.

**Comment by riekusr** [comment]
Found it in core.js thanks! Also trying the upset node on the latest version later today

**Comment by tomkerswill** [comment]
This looks like it's still an issue --- so all requests result in a 404. Do you have any time to update the code to the new version of the API. Thanks -- looks like this will be a great tool!

**Comment by the-greenman** [comment]
Hey Tom.

Not too much chance to look at this.
I've not needed to look at codas api a while.

I'll need to review the current state of the API and see if we can update
things.

Peter
